csrankings.csv

```diff
@@ -13760,6 +13764,7 @@ Nivio Ziviani,UFMG,http://homepages.dcc.ufmg.br/~nivio,OZ2ju_EAAAAJ
 Nizamettin Aydin,Yıldız Technical University,https://www.ce.yildiz.edu.tr/personal/naydin?lang=en,JhyQcSIAAAAJ
 Noa Agmon,Bar-Ilan University,http://u.cs.biu.ac.il/~agmon,arDFD58AAAAJ
 Noah A. Smith,University of Washington,https://www.cs.washington.edu/people/faculty/nasmith,TjdFs3EAAAAJ
+Noah D. Goodman,Stanford University,https://cocolab.stanford.edu/ndg.html,OUpIbcQAAAAJ

```

Comment:
Uber.

Reply:
ok. Since Uber AI Lab no longer exists, I don't know whether this is still the case.

Reply:
Probably not, but please check.